dao/src/main/java/greencity/repository/LocationRepository.java

```diff
@@ -3,5 +3,16 @@
 import greencity.entity.Location;
 import org.springframework.data.jpa.repository.JpaRepository;
 
+import java.util.Optional;
+
 public interface LocationRepository extends JpaRepository<Location, Integer> {
```

Comment:
Please, use Long (Int64) instead if Integer (Int32), otherwise it will cause errors in future.